src/Forms/Shared/Samples/MapView/DisplayGrid/DisplayGrid.xaml.cs

```diff
@@ -0,0 +1,141 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using System;
+using Xamarin.Forms;
+using Colors = System.Drawing.Color;
+
+namespace ArcGISRuntime.Samples.DisplayGrid
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Display a grid",
+        "MapView",
+        "Display and work with coordinate grid systems such as Latitude/Longitude, MGRS, UTM and USNG on a map view. This includes toggling labels visibility, changing the color of the grid lines, and changing the color of the grid labels.",
+        "Choose the grid settings and then tap 'Apply settings' to see them applied.")]
+    public partial class DisplayGrid : ContentPage
+    {
+        public DisplayGrid()
+        {
+            InitializeComponent();
+
+            Title = "Display a grid";
+
+            // Create the UI, setup the control references and execute initialization
+            Initialize();
+        }
+
+        private void Initialize()
+        {
+            // Set up the map view with a basemap.
+            MyMapView.Map = new Map(Basemap.CreateImageryWithLabelsVector());
```

Comment:
Since there's still an issue displaying vector tiles on Android emulators, should we use raster tiles for Android/Forms samples?